src/base/bittorrent/ltqhash.h

```diff
@@ -34,14 +34,8 @@
 
 #include <QHash>
 
-namespace libtorrent
+template <typename T, typename Tag>
+uint qHash(const libtorrent::aux::strong_typedef<T, Tag> &key, const uint seed)
```

Comment:
From Qt documentation:
>A QHash's key type has additional requirements other than being an assignable data type: it must provide operator==(), and there must also be a _**qHash() function in the type's namespace**_ that returns a hash value for an argument of the key's type.

Reply:
OK, false-positive from clazy. Reverted.

Reply:
>OK, false-positive from clazy. Reverted.

Maybe it's worth adding a comment to avoid repeated attempts to "fix" this code in the future?

Reply:
>Maybe it's worth adding a comment to avoid repeated attempts to "fix" this code in the future?

Maybe it is best to leave it as is.
This turned out to be more complicated than I thought. That part is from Qt5 doc and Qt6 doc has removed it: https://doc.qt.io/qt-6/qhash.html#the-hashing-function
Furthermore, Qt6 will also lookup `std::hash` when it is available. However IMO `std::hash` provided from libtorrent is less optimal as it is missing the `seed` parameter, i.e. I mean we probably should keep using the current implementations.

Reply:
>Qt6 doc has removed it

It's just reworded:

>The first way is by having an overload of `qHash()` in K's namespace.